src/code/bank1.asm

```diff
@@ -1257,102 +1257,172 @@ jr_001_5C7B::
 label_001_5D13::
     ret                                           ; $5D13: $C9
 
-Data_001_5D14::
-    db  $98, $CB, $06, $7E, $7E, $7E, $7E, $7E, $7E, $7E ; $5D14
-    db  $98, $EB, $06, $7E, $7E, $7E, $7E, $7E, $7E, $7E ; $5D1E
-    db  $00                                       ; $5D28
-
-Data_001_5D29::
-    db  $99, $2B, $06, $7E, $7E, $7E, $7E, $7E, $7E, $7E ; $5D29
-    db  $99, $4B, $06, $7E, $7E, $7E, $7E, $7E, $7E, $7E ; $5D33
-    db  $00                                       ; $5D3D
-
-Data_001_5D3E::
-    db  $99, $8B, $06, $7E, $7E, $7E, $7E, $7E, $7E, $7E ; $5D3E
-    db  $99, $AB, $06, $7E, $7E, $7E, $7E, $7E, $7E, $7E ; $5D48
-    db  $00                                       ; $5D52
+SaveSlot1HeartsDrawData::
+    ; First row:
+    db  $98 ; wDrawCommand.destinationHigh
+    db  $CB ; wDrawCommand.destinationLow
```

Comment:
Nitpick, and really not important: maybe 

```suggestion
    dw  $98CB ; wDrawCommand.destination
```

?

Reply:
Those comments refer to the corresponding labels here, and there is no `destination` label: https://github.com/zladx/LADX-Disassembly/blob/main/src/constants/memory/wram.asm#L2876

Also, the Game Boy is little-endian, even though this struct is big-endian for some reason, so calling it `destination` would perhaps also confuse people who expect to be able to use the whole thing as an address? :)